src/runtime/audience-action-local-flow.ts

```diff
@@ -352,9 +362,13 @@ export class AudienceActionLocalFlow implements AudienceActionFlow {
     this.deps_.callbacks().triggerLoginRequest({linkRequested: false});
   }
 
-  private async complete_() {
+  private async fetch_(
+    method: string,
+    endpoint: string,
+    queryParams: string[][]
+  ): Promise<Response> {
     const init: RequestInit = {
-      method: 'POST',
+      method,
       headers: {
```

Comment:
Out of curiosity, do we need to manually set headers like this? I wonder if we can use (or extend) the existing method provided by fetcher.ts

Reply:
I didn't wanna use fetcher for GET because it would hide the response, making it harder to catch errors for my logging. Didn't use it for POST for consistency, but I think we could use it.

Reply:
Would you elaborate more on "GET would hide the response".
I wonder if we could either
- Provide a get method from the fetcher interface
- Or at least put header defined in the centralized place 

It will ensure all the request specs are consistently updated in the future.